### ISSUE TITLE
Scrobbling on last.fm not working

### ISSUE THREAD

**Issue by tmldkge** [SpotX-Official/SpotX#304]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

Poland

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.2.4.907.g126451d5
- Windows: Windows 10 Pro 21H2 19044.2006
- PowerShell: 5.1.19041.1682
```

### 📝 Description

Basically it stopped working on 31st january at night, my last scrobble is from 1:30 am. I have been using an old version of spotify and SpotX for long time (93rd version, not sure which one it was in that google spreadsheet with lots of versions). I have uninstalled Spotify, then downloaded newest version of it and SpotX and it's still not working so I'm looking for help here.

**Comment by amd64fox** [comment]
Reconnect your Spotify account to last.fm, or downgrade to an older version of Spotify, Spotx has no effect on last.fm scrobling or discord status
